### PR TITLE
design teardown project

### PR DESCRIPTION
**Pull request by maosan132**
This project was created with flexbox and CSS-grid. Colors used grayscale shadows. The typography used: Roboto. The project was filled with dummy text.

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -1,13 +1,320 @@
...
+
+<body>
+    <header>
+        <ul>
+            <li class="item1 logo">
+                <a href="#">
+                    Smashing
+                    Magazine
+                </a>
+            </li>
```
Comment (addod19) [inline index.html]: It uses ```<div>```  to position elements as at the source page
k1

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi, @maosan132 and @chinweokwu you have done a great job👍 . However, we need to make a few changes to get the project approved. Find comment below:
- We are currently missing the remaining 2 rows
check
```Currently``` we are missing additional 2 row which has 3 articles each taking 1/3 width
missing
And this
missing1
- Footer contains center links
fl
- Kindly center this text
opt

Good luck with your next submission
Happy coding 💻 
Daniel Larbi Addo

**Comment by maosan132** [comment]
Thank you for your kind answer. We didn't understand these points:


   - Footer contains center links
   - This part does not look like 2 rows. Kindly let it look like 2 rows.
   - It uses <div> to position elements as at the source page

We look forward for your reply,

Mauricio and Paul

**Comment by SofiBretz** [comment]
### **Changes required ♻️**

### **Hello team 🌟**
Great work on this project, but you need to fix the issues below before moving on ⬇️

- [ ] The Stickler didn't pass, please fix the stickler errors until you didn't get any errors (This project can not be approved if you have stickler errors). 

- [ ] Consider using a `div` tag on your navBar. Please use the `div` in all your navBar, not only for the `class= "logo` 



- [ ] Please follow the structure of the original page on the project, for example, after the header, it's supposed to have 3 columns, and in your project, you used 2 columns and the same with the next other structure of the page, remember that follow the columns at the original page.




[OPTIONAL]

- [ ] On `reset.css` file does not contain info, so you can delete it.

- The comments on your commits are very clear, very good work.
- Very good that you use the readme template of Microverse.
- Very well that you work in a branch that is not master.

_**After making the required changes please submit a new code review request.**_

Good luck! 🍀 You are almost there! 💪  

SofiBretz.

File: index.html [hunk 2]
```diff
@@ -1,13 +1,374 @@
...
+
+            <li class="item1">
+                <a href="#">
+                    <img src="https://d33wubrfki0l68.cloudfront.net/2bc30327ebf529a249e825447d86f18546a1d6db/36b19/images/nav-icons/articles.svg"
+                        alt="" width="39" height="44">
+
+                    <h2>Articles</h2>
+                    <p>Lorem Ipsum </p>
+                </a>
+            </li>
```
Comment (addod19) [inline index.html]: wrap this ```<li>``` and the other ```<li>``` with a ```<div>```. Example ```<div> <li>code here</li></div>```

Comment (maosan132) [reply index.html]: already done!

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team       
#### Great Job so far, but there are a few more things required. 
### General
- [ ] Let the articles be positioned well  like the original
#### Your's


#### Original


- [ ] This should have one article, not three.
#### Your's



### Make changes and submit another code review request.
*Happy coding*

File: index.html [hunk 3]
```diff
@@ -1,13 +1,373 @@
...
+                    Smashing
+                    Magazine
+                </a>
+            </div>
+            <div class="nav">
+                <ul>
+                    <li>
+                        <a href="#">
+                            <img src="https://d33wubrfki0l68.cloudfront.net/2bc30327ebf529a249e825447d86f18546a1d6db/36b19/images/nav-icons/articles.svg"
+                                alt="" width="39" height="44">
```
Comment (euqueme) [inline index.html]: remove the selectors width and height in line #28 as no inline styling should be in the HTML file

File: index.html [hunk 4]
```diff
@@ -1,13 +1,373 @@
...
+
+                            <h2>Articles</h2>
+                            <p>Lorem Ipsum </p>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="https://d33wubrfki0l68.cloudfront.net/2bc30327ebf529a249e825447d86f18546a1d6db/36b19/images/nav-icons/articles.svg"
+                                alt="" width="39" height="44">
```
Comment (euqueme) [inline index.html]: remove the selectors width and height in line #37 as no inline styling should be in the HTML file

File: index.html [hunk 5]
```diff
@@ -1,13 +1,373 @@
...
+                            <h2>Articles</h2>
+                            <p>Lorem Ipsum </p>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="https://d33wubrfki0l68.cloudfront.net/2bc30327ebf529a249e825447d86f18546a1d6db/36b19/images/nav-icons/articles.svg"
+                                alt="" width="39" height="44">
```
Comment (euqueme) [inline index.html]: remove the selectors width and height in line #45 as no inline styling should be in the HTML file

File: index.html [hunk 6]
```diff
@@ -1,13 +1,373 @@
...
```
Comment (euqueme) [inline index.html]: remove the selectors width and height in line #53 as no inline styling should be in the HTML file

File: index.html [hunk 7]
```diff
@@ -1,13 +1,373 @@
...
```
Comment (euqueme) [inline index.html]: remove the selectors width and height in line #61 as no inline styling should be in the HTML file

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @maosan132 and @chinweokwu  :handshake: 

### Your project looks great, you've done a great work so you should feel very proud of it :star2:  thank you for doing all the changes requested by previous reviewers  :medal_sports: 

### I have no changes in the style left to point out, but you need to address some mandatory requirements yet :point_down: 

## General

- [ ] Check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload), we can only approve projects with no HTML errors

image

- [ ] Please remove all styling selectors from your HTML file as no inline styling is required for the approval (I have left comments in your code where inline styling was spotted, please remove them from the HTML and only use ID's or classes for the styling in the HTML file)

### That's all, team! :raised_hands: 

### Please :pray: do the changes :memo: and request for a code review when you are done :rocket: 

Happy coding! :woman_technologist: 

Any question? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team       
#### Great Job so far, but there is one more thing required,you have to addresse each error step by step on the validator site. 
### General
- [ ] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload


### Make changes and submit another code review request.

*Happy coding*

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Hi Team,
## Great  work, Congratulation :+1: 
You did all changes required previously. Go ahead and merge to master
_Goodluck in your next project_ *Happy coding*

**Pull request merged**